tests/integration/attachments.test.js

```diff
@@ -45,7 +45,6 @@ describe("Tests for uploading/deleting attachments through API calls - in-memory
     );
 
     try {
-      //trigger to upload attachment
```

Comment:
why have we removed all the comments here?

Reply:
I have updated the new commit with all the previously present comments.